src/app/onboarding/views/DataplanesOverview.vue

```diff
@@ -1,23 +1,31 @@
 <template>
-  <RouteView>
+  <RouteView
+    v-slot="{ t }"
+  >
     <RouteTitle
       :title="t('onboarding.routes.dataplanes-overview.title')"
     />
     <AppView>
       <OnboardingPage>
         <template #header>
-          <OnboardingHeading>
-            <template #title>
-              <p>{{ title }}</p>
-            </template>
-
-            <template
-              v-if="description !== null"
-              #description
-            >
-              <p>{{ description }}</p>
-            </template>
-          </onboardingheading>
+          <template
+            v-for="item in [
+              tableData.data.length > 0 ? 'success' : 'waiting'
+            ]"
```

Comment:
Stylistically, pretty sure we decided that this style is fine, for the moment at least. There are plenty of places where we've used this style, and its not just code I've added/changed. I think the phrase was "having the data where its used", or similar to that. I'm resolving this one, unless I hear more.

Reply:
For the sake of reference: Computed properties are in every way better to handle this.

The other cases where we use this style aren’t comparable because they’re ones where we use DataSource which introduces `data` in a template-scoped manner (i.e. we would need to set a `ref` using `update` and add a computed property on top).

I don’t think it’s really an advantage to have this variable be more narrowly scoped to where it’s used. The component being the scoping boundary for computed properties is almost always completely fine and I’m not aware of any bugs that were introduced by us being able to read a component-scoped variable when we shouldn’t have.

I don’t understand the reluctance to use the core building blocks of Vue (specifically `computed`).

Reply:
There's no reluctance, I don't have a problem using it where it helps, and whilst I don't want to speak for others I think this is echoed by other folks on the team.

I think we've discussed at length that in these simple cases its probably not helping, especially when we are using lots for very simple view logic that in some cases barely changes.

I think you can probably equate this to an argument between:

```javascript
const { title, description, message} = { title: 'Title', description: 'Description', message: 'Message'}
console.log(title)
// vs
const title = 'Title'
const description = 'Description'
const message = 'Message'
console.log(title)
```

Apart from the second example would probably be more a realistic comparison like:

```javascript
console.log(title)

// loads of code
// loads of code
const title = 'Title'
// loads of code
const description = 'Description'
// loads of code
const message = 'Message'
// loads of code
// loads of code
```

So there's one way at least where I think it's worse